styles.css

```
@@ -2,4 +2,41 @@ body {
     background-color: lightseagreen;
 }
 
+.demo{
+    display: flex;
+}
+
+.demo-image img{
+    height: auto;
+    width: 55em;
+    display: inline-block;
+}
+
+.example-circles{
+    background-color: palegoldenrod;
+    height: auto;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.outer-circle{
+    height: 14em;
+    width: 14em;
+    border: 60px solid white;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-shadow: 1px 1px 2px lightgrey, 0 0 25px grey, 0 0 5px lightgrey;
+}
+
+.inner-circle{
+    height: 7em;
+    width: 7em;
+    border: 13px solid white;
+    border-radius: 50%;
+    box-shadow: 1px 1px 2px lightgrey, 0 0 25px grey, 0 0 5px lightgrey;
```

I love the shorthand syntax here! I think it'd look a little closer to the comp if instead of a grey box shadow it was more yellowish though. Maybe something like:

```suggestion
    box-shadow: 1px 1px 2px #fffefb, 0 0 25px #fbda61, 0 0 5px #ffbc00;
```

And mess with the colors and opacity to your liking! Don't forget to use the dev tools to inspect, it's a great quick and easy way to set colors and try different values out!